src/pages/Order/Order.js

```diff
@@ -42,16 +104,33 @@ const Order = () => {
         </PriceBox>
         <MethodTitle>결제 수단</MethodTitle>
         <OrderMethodBox>
-          <OrderMethod>보유 캐시</OrderMethod>
-          <OrderMethodBalance>
-            {userBalance.toLocaleString('ko-KR')}원
-          </OrderMethodBalance>
+          {ORDER_METHOD.map(({ id, method }) => {
+            return (
+              <OrderMethodLabel key={id}>
+                <SelectMethod
+                  type="radio"
+                  value={method}
+                  name="method"
+                  onChange={e => {
+                    selectOrderMethod(e);
+                    setOrderMethod(e.target.value);
+                  }}
```

Comment:
위 리뷰를 참고하면 이 부분은 단순히 method값만 state에 저장하면 될 것 같습니다.
```suggestion
                  onChange={() => setOrderMethod(method)}
```